openpype/pipeline/create/creator_plugins.py

```diff
@@ -611,6 +615,56 @@ def get_pre_create_attr_defs(self):
         """
         return self.pre_create_attr_defs
 
+    def get_transient_data_profile(self, instance, task_type=None):
```

Comment:
I find it a bit confusing that they are called exactly the same :upside_down_face: